src/components/FeatureViews/Scheduler/DisplaySessionView.js

```diff
@@ -367,7 +370,11 @@ const DisplaySessionView = ({ course, session, handleToggleEditing }) => {
                     <Button color="primary" onClick={handleEditToggle(true)}>
                         Cancel
                     </Button>
-                    <Button color="primary" onClick={handleEditToggle(false)}>
+                    <Button
+                        color="primary"
+                        component={NavLink}
+                        to={{"pathname":`/scheduler/edit-session/${course.course_id}/${session.id}/${instructor_id}/edit`
+                        ,"state":{course:course, session:session}}}>
```

Comment:
You can't use `state` on links for a necessary parameter, it will not be loaded if you load the link by copy pasting into a new tab, for example. Revert to URL parameters.